src/Pulse.php

```diff
@@ -129,18 +129,16 @@ public function register(array $recorders): self
             ->filter(fn ($recorder) => $recorder->listen ?? null)
             ->each(fn ($recorder) => $event->listen(
                 $recorder->listen,
-                fn ($event) => $this->rescue(fn () => Collection::wrap($recorder->record($event)))
+                fn ($event) => $this->rescue(fn () => $recorder->record($event))
```

Comment:
`Collection::wrap` is a remnant from when recorders used to return entries. They no longer do.